DataFormats/HGCalDigi/src/classes_def.xml

```diff
@@ -46,19 +46,19 @@
   <class name="hgcaldigi::HGCalDigiHost"/>
 
   <class name="hgcaldigi::HGCalDigiSoALayout<128,false>"/>
-  <class name="edm::Wrapper<PortableHostCollection<hgcaldigi::HGCalDigiSoALayout<128,false> > >" splitLevel="0"/>
+  <class name="edm::Wrapper<PortableHostCollection<hgcaldigi::HGCalDigiSoALayout<128,false> > >" splitLevel="0" rntupleStreamerMode="true"/>
```

Comment:
Do we need the `rntupleStreamerMode` for the `edm::Wrapper`?

Or is the dictionary for `PortableHostCollection<hgcaldigi::HGCalDigiSoALayout<>>` missing?

Reply:
The `PortableHostCollection<hgcaldigi::HGCalDigiSoALayout<>>` is missing and it isn't my code :)